Gemfile

```diff
@@ -31,6 +31,9 @@ gem 'sentry-ruby'
 gem 'sentry-rails'
 gem 'rouge'
 gem 'diff-lcs'
+gem 'clipboard'
+gem 'jquery-turbolinks'
+gem "ffi", :platforms => [:mswin, :mingw]
```

Comment:
These gems won't work for what you're trying to do, I'm afraid. The clipboard gems gives you access to the clipboard in ruby code, but ruby runs *on the server*. If you run `<%= Clipboard.copy(...) %>`, this will trigger the clipboard on the server that is running the code and not on the browser. I'll show you a working example elsewhere.